Bottom: cf25db33869ff9289ec152ca8e452de147e83c1b
Top:    72274864ab2fa3553d2cdfb306dae420573ff3ee
Author: scott snyder <sss@karma>
Date:   2020-02-22 23:57:45 -0500

Updates for GirlsWithSlingshots.

Switched to https; another block of missing IDs.



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
index b94476e..f036506 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
@@ -31,11 +31,11 @@ public class GirlswithSlingshots extends IndexedComic {
 
 	@Override
 	protected String getFrontPageUrl() {
-		return "http://www.girlswithslingshots.com/";
+		return "https://www.girlswithslingshots.com/";
 	}
 
 	public String getComicWebPageUrl() {
-		return "http://www.girlswithslingshots.com/";
+		return "https://www.girlswithslingshots.com/";
 	}
 
 	@Override
@@ -64,7 +64,8 @@ public class GirlswithSlingshots extends IndexedComic {
 	@Override
 	public int addException(int id, int increment) {
 		while(true) {
-			if(!mHash.containsKey(id)) {
+			if(!mHash.containsKey(id))
+                        {
 				break;
 			}
 			id += increment;
@@ -75,7 +76,7 @@ public class GirlswithSlingshots extends IndexedComic {
 	@Override
 	public String getStripUrlFromId(int num) {
 //Log.d("GWS", "num " + num);
-		return "http://www.girlswithslingshots.com/comic/gws-chaser-" + num;
+		return "https://www.girlswithslingshots.com/comic/gws-chaser-" + num;
 	}
 
 	@Override
